verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx

```diff
@@ -182,8 +185,8 @@ describe('<SignUpForm />', () => {
             />,
         );
 
-        userEvent.type(screen.getByLabelText('Password'), '12345');
-        userEvent.type(screen.getByLabelText(/Repeat password/), '6789');
+        userEvent.type(screen.getByLabelText('Password *'), '12345');
+        userEvent.type(screen.getByLabelText(/Repeat password */), '6789');
```

Comment:
sorry there are multiple instances of this same problem, only commented on the first